packages-native/safe-area-view/src/SafeAreaView.tsx

```diff
@@ -9,11 +9,7 @@ export const SafeAreaView = (props: SafeAreaViewProps<SafeAreaViewStyle>): JSX.E
     const customStyles = props.style.filter(o => o != null);
     const styles = flattenStyles(defaultSafeAreaViewStyle, customStyles);
 
-    return (
-        <Fragment>
-            <ReactSaveAreaView style={{ flex: 0, backgroundColor: styles.unsafeAreaTop.backgroundColor }} />
-            <ReactSaveAreaView style={{ ...styles.container, flex: 1 }}>{props.content}</ReactSaveAreaView>
-            <ReactSaveAreaView style={{ flex: 0, backgroundColor: styles.unsafeAreaBottom.backgroundColor }} />
-        </Fragment>
-    );
+    const overflowStyleFix: ViewStyle = { flexDirection: "row", flex: 1, alignItems: "stretch", overflow: "hidden" };
+
+    return <ReactSaveAreaView style={{ ...styles.container, ...overflowStyleFix }}>{props.content}</ReactSaveAreaView>;
```

Comment:
style={[styles.container, overflowStyleFix]}

Reply:
Changed it.